qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -598,19 +597,6 @@ class ScalableSymbolicPulse(SymbolicPulse):
     :math:'\text{amp}\times\exp\left(i\times\text{angle}\right)' is compared.
     """
 
-    @deprecate_arg(
```

Comment:
Removal of deprecated code in this file is done by https://github.com/Qiskit/qiskit/pull/11403, please revert this change not to conflict.